release_tools/changelog.py

```diff
@@ -208,16 +211,36 @@ def create_changelog_entry_content(title, category, author=None, issue=None,
     else:
         content = stream
 
+    if not content:
+        msg = "Aborting due to empty entry content"
+        raise click.ClickException(msg)
+
+    try:
+        data = yaml.safe_load(content)
+    except yaml.YAMLError as exc:
+        pm = exc.problem_mark
+        msg = "File has an issue on line {} at position {}".format(pm.line, pm.column)
+        raise click.ClickException(msg)
```

Comment:
All of this should be part of a different function to separate the logic.

Reply:
Yeah, makes much sense.